content/en/docs/toolbox/auditing-operating.md

```diff
@@ -0,0 +1,385 @@
+
+#### Rule condition
+
+`Condition` is a filtering expression that can use comparison operators (=, !=, <, <=, >, >=, contains, in, like, regex) and can be combined using Boolean operators (and, or and not) and parentheses. Here are the supported Filters.
+
+ Filter | Description
+ ---    | --- 
+ Workspace                | The workspace which this audit event happened
+ Devops                   | The devops project which this audit event happened
+ Level                    | Level of auditing logs
+ RequestURI               | RequestURI is the request URI as sent by the client to a server.
+ Verb                     | Verb is the verb associated with the request
+ User.Username            | The name that uniquely identifies this user among all active users.
+ User.Groups              | The names of groups this user is a part of.
+ SourceIPs                | Source IPs, from where the request originated and intermediate proxies.
+ ObjectRef.Resource       | The resource of the object associated with the request.
+ ObjectRef.Namespace      | The namespace of the object associated with the request.
+ ObjectRef.Name           | The name of the object associated with the request.
+ ObjectRef.Subresource    | The subresource of the object associated with the request.
+ ResponseStatus.code      | Suggested HTTP return code for the request
+ ResponseStatus.Status    | Status of the operation.
+ RequestReceivedTimestamp | Time the request reached the apiserver.
+ StageTimestamp           | Time the request reached the current audit stage.
+
+ For example, to match all logs in the namespace `test`,
+
+```
+ObjectRef.Namespace = "test"
+```
+
+ to match all logs in the namespaces start with `test`,
+
+```
+ObjectRef.Namespace like "test*"
+```
+
+ to match all logs happened in the latest one hour,
+
+```
+RequestReceivedTimestamp >= "2020-06-12T09:23:28.359896Z" and RequestReceivedTimestamp <= "2020-06-12T10:23:28.359896Z"
+```
+
+#### Macro
+`Macro` is a rule condition snippets that can be re-used inside rules and even other macros. Macros provide a way to name common patterns and factor out redundancies in rules. Here¡¯s an example of a macro.
+
+```
+apiVersion: auditing.kubesphere.io/v1alpha1
+kind: Rule
+metadata:
+  name: alerting-rule
+  labels:
+    workspace: system-workspace
+    type: alerting
+spec:
+  rules:
+  - name: pod
+    type: macro
+    desc: pod
+    macro: ObjectRef.Resource="pods"
+```
+
+ > `Macro` can bu used in rules or other macros like ${pod} or ${alerting-rule.pod}. The difference between these two methods is, ${pod} only can be used in the CRD Rule `alerting-rule`, ${alerting-rule.pod} can be used in all CRD Rule. It is also applied to lists and alias.
```

Comment:
->
`Macro` can be used in rules or other macros like ${pod} or ${alerting-rule.pod}. The difference between these two methods is that ${pod} only can be used in the CRD Rule `alerting-rule`, while ${alerting-rule.pod} can be used in all CRD Rules. It also applies to lists and alias.